graphql/src/query.rs

```diff
@@ -18,6 +18,18 @@ pub struct Query;
 
 #[async_graphql::Object]
 impl Query {
+    // query for 1 collection based on slug, and make a collection object with all the stuff in it.
+    async fn get_edited_collection(
+        &self,
+        context: &Context<'_>,
+        slug: String,
+    ) -> FieldResult<Option<Vec<EditedCollection>>> {
```

Comment:
The return type should be singular, right? You're querying for a single collection, not several of them. Something like `FieldResult<Option<EditedCollection>>`